nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -31,6 +31,10 @@ In addition to numerous new and upgraded packages, this release has the followin
   The `nimPackages` and `nim2Packages` sets have been removed.
   See https://nixos.org/manual/nixpkgs/unstable#nim for more information.
 
+- `lib.options.mkPackageOptionMD` is now deprecated. It is an alias of
+  `lib.options.mkPackageOption`, and any invocations should be changed
+  to use it's non `MD` equivalent.
+
```

Comment:
I don't think it makes sense to deprecate something without a warning. I'd rather not have anything in the release notes for now.

```suggestion
```

Reply:
Removed last commit